src/modules/vtol_att_control/vtol_type.h

```diff
@@ -216,6 +222,8 @@ class VtolType
 
 	motor_state _motor_state = motor_state::DISABLED;
 
+	float _hover_pusher_or_tilt_forward_actuation = 0.0f; // normalized pusher throttle (standard VTOL) or tilt (tiltrotor)
```

Comment:
@sfuhrer I would prefer this to remain a forward throttle quantity. Tiltrotor can then convert it to a tilt actuator command.